utility/utils.py

```diff
@@ -607,11 +607,13 @@ def get_latest_container(version):
         Container details dictionary with given format:
         {'docker_registry': docker_registry, 'docker_image': docker_image, 'docker_tag': docker_tag}
     """
-    url = "http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/latest-RHCEPH-{}.json".format(
```

Comment:
This URL looks Obsolete so added - http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/